tests/moxa1210.py

```diff
@@ -55,3 +66,12 @@ def test_WHEN_DI_input_is_switched_on_THEN_only_that_channel_readback_changes_to
                 continue
 
             self.ca.assert_that_pv_is("CH{:02d}:DI".format(test_channel), "Low")
+
+    @parameterized.expand([
+        ("CH{:02d}:DI:CNT".format(channel), channel) for channel in CHANNELS
+    ])
+    def test_WHEN_di_input_is_triggered_THEN_di_counter_increases(self, channel_pv, channel):
+        self.resetDICounters()
+        self._lewis.backdoor_run_function_on_device("set_di", (channel, (True,)))
+
+        self.ca.assert_that_pv_is(channel_pv, 1)
```

Comment:
Would it be useful to demonstrate that the counters are properly adding up over multiple switches by turning a DI on and off again a few times?

Reply:
Created a new test to check that individual channels would correctly log multiple events.